src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -965,21 +979,14 @@ public void Connect(IPAddress[] addresses, int port)
                         lastex = null;
                         break;
                     }
-                    catch (Exception ex)
+                    catch (Exception ex) when (!ExceptionCheck.IsFatal(ex))
```

Comment:
Ok by me.  Separately, do you think these should be deleted?  If so can we file an issue?